docs/wiki/framework/events-framework.md

```diff
@@ -81,7 +81,7 @@ MenuType: 0 = Interaction, 1 = Self Interaction
 
 | Event Key | Parameters | Locality | Type | Description |
 |----------|---------|---------|---------|---------|---------|
-|`ace_allowDefuse` | [_mine, _allow] | Global | Callable | Exclude or Include a mine from the dynamic defuse action
+|`ace_allowDefuse` | [_mine, _allow] | Global | Callable | Set allowment of the dynamic defusal action on a mine
```

Comment:
I listed it as Global as this is how it should be called to get the "standard" effect. No other action is like this one right now, so I don't know if this is correct. All other actions that are callable are targeted at locality not pick-and-choose like this one.

Reply:
Maybe do "Any" or "Global/Target".

Reply:
"Global/Target", that's most appropriate I'd say. changing now.